### ISSUE TITLE
Is there a method for declaring sinks directly in the Jimple form inside FlowDroid

### ISSUE THREAD

**Issue by PhoenixMallu** [secure-software-engineering/FlowDroid#507]
Dear Steven

Can we declare sinks directly in to the Jimple body of a code so that the issink() method of FlowDroid correctly maps the information flow from source to such Jimple declared sinks.

**Comment by StevenArzt** [comment]
You want to have an individual statement as a sink? Have a look at the ``StatementSourceSinkDefinition``.

**Comment by PhoenixMallu** [comment]
@StevenArzt  yes I want to add an "if condition" as sink... how should I mark the sink in Jimple so that it can be identified by FlowDorid.. Where should I declare such sinks in FlowDroid? In cases involving statements as sinks will it affect the access path?
